### PR TITLE
AAE-17000 Upgrade Activiti to Spring Boot 3.2

### PR DESCRIPTION
**Pull request by igdianov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -936,6 +944,14 @@ limitations under the License.
...
   </profiles>
   <repositories>
+    <repository>
+      <id>spring-milestones</id>
+      <name>Spring Milestones</name>
+      <url>https://repo.spring.io/milestone</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```
Comment (atchertchian) [inline pom.xml]: Same than https://github.com/Activiti/activiti-cloud/pull/1224#discussion_r1418776842: I wonder if it should be moved after central (?) cc @matteo4diani

Comment (igdianov) [reply pom.xml]: We don't need to use Spring Milestones repository after official SB release has been published  in Maven Central, so I will remove it.

Comment (atchertchian) [reply pom.xml]: Ah! Makes more sense :+1:

**Review by andrea-ligios — APPROVED** [review]

**Review by graric — APPROVED** [review]

**Review by erdemedeiros — APPROVED** [review]
LGTM!
I removed the reference to Spring Milestones

**Pull request merged**